docs/evaluation_plan.md

```diff
@@ -0,0 +1,128 @@
+We will decompose WIS into dispersion, overprediction, and underprediction as well as compute coverage metrics for all models, local and aggregate.
+Relative WIS will be computed relative to the corresponding aggregate-level model.
+Values of 1 indicate equivalent forecast performance, values of less than 1 indicate improved performance, and greater than 1 indicate improved performance compared to the aggregate level version of the model.
+
+We examine the relative and absolute WIS across multiple stratifications: overall, by nowcast horizon, and by location.
+
+#### Model-based evaluation of aggregate vs local forecasts
+
+Similar in spirit to Sherratt et al., we will perform a model-based evaluation to account for confounding variables impacting the performance of the local model compared to the aggregate model.
+
+To assess the impact of local forecasting on forecast performance, we will include the following confounding variables:
+- score of the corresponding aggregate model
+- forecast date and location together
+- location
+- model
+
+Resulting in the following model formulation:
+```math
+WIS^{local}_{h,d,l,m} \sim \beta + WIS^{aggregate}_{h,d,l,m} + s(location, bs = "re") + s(forecastdate, by = "location") + s(model, bs = "re")
+```
+Where $h$ is the forecast horizon (from -1 to 4 weeks), $d$ is the forecast date, $l$ is the location of the forecast (the borough or metro area), and $m$ is the model.
+The $WIS^{aggregate}_{h,d,l,m}$ term should roughly account for the average forecast difficulty of a particular model at that particular forecast location, forecast date, and horizon.
+
+We can roughly interpret $\beta$ as the average effect of local forecasting on forecast performance compared to the aggregate forecast, and the $s(model, bs = "re")$ term as the average additional effect across all locations, forecast dates, and horizons from a particular model.
+
+The $s(location, bs = "re")$ term tells us about the additional effect of location on the relative value of local forecasting, whereas the $s(forecastdate, by = location)$ term tells us about the effect of the forecast date and location combinations on the relative value of local forecasting.
+
+We will plot the partial effects of each of these components in order to understand the drivers of differences in forecast performance between local and aggregate forecasts.
+
+### Retrospective analysis: local model comparison
+This analysis will compare performance between the different retrospectively produced local forecasts.
+
+We will use the scores from the previous section, focusing only on the local scores.
+Relative WIS will be computed relative to the baseline local model.
+
+Again, we will examine the relative and absolute WIS scores across multiple stratifications: overall, by nowcast horizon, and by location.
+
+#### Model-based evaluation for comparing across local models
```

Comment:
Might not really be necessary for this retrospective analysis because all models are forecasting everywhere for all forecast dates